recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
```

Comment:
Just clarifying: why limit Linux and Mac on Python version?  Sure, people will complain about Windows, but that's just what we've got.